src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
I've gotten a bit excitable and just rewritten the thing... Can you try building the issue105 branch and see if what's there fixes what you're seeing?